pipelines/jobs/Semeru_configuration/jdk11u_pipeline_config.groovy

```diff
@@ -177,7 +177,7 @@ class Config11 {
         aarch64Linux    : [
             os                  : 'linux',
             arch                : 'aarch64',
-            dockerImage         : 'adoptopenjdk/centos7_build_image',
+            dockerImage         : 'ghcr.io/adoptium/adoptium_build_image:centos7',
             dockerNode         : 'sw.tool.docker',
             dockerCredential    : '9f50c848-8764-440d-b95a-1d295c21713e',
```

Comment:
These will have to be updated from DockerhHub cred to GitHub cred.
`f5a0bd2f-093e-41ea-bd6f-875936334a63`